apps/root/src/services/earnService.ts

```diff
@@ -301,6 +301,15 @@ export class EarnService extends EventsManager<EarnServiceData> {
     this.allStrategies = allStrategies;
   }
 
+  batchUpdateUserStrategies(userStrategies: SdkEarnPosition[]) {
+    let storedUserStrategies = [...this.userStrategies];
+
+    userStrategies.forEach((strategy) => {
+      storedUserStrategies = this.updateUserStrategy(strategy, false);
```

Comment:
if its not updating the store, this is only going to apply for the last strategy passed here, since updateUserStrategy is going to extend from this.userStrategies